src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -89,6 +89,9 @@ object SamResourceActions {
   val addChild = ResourceAction("add_child")
   val removeChild = ResourceAction("remove_child")
   val listChildren = ResourceAction("list_children")
+  val adminReadPolicies = ResourceAction("admin_read_policies")
```

Comment:
I realized we never updated the Sam model. I think this is what we need? @marctalbott does this look right to you?

Reply:
Yeah, this is correct, we need this to perform the auth checks that you added in the routes

Reply:
I saw this too and had this in my branch, good catch